pkg/version/windows_versioninfo.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
The intent of this file is to act as a template that gets generated at build-time, right?